pkg/apiserver/api/v1/provider.proto

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+
+import "types.proto";
+
+message Provider {
+  string name = 1;
+  string tenant = 2;
+  AccountMetadata metadata = 3;
+}
+
+message ProviderList {
+  repeated Provider items = 1;
+  string continue = 2;
+}
+
+message ProviderRequest {
+  string name = 1;
+  string tenant = 2;
+}
+
+message ProviderListRequest {
+  string tenant = 1;
+  string labelSelector = 2;
+  int64 limit = 3;
+  string continue = 4;
+}
+
+
+message AccountMetadata {
+  string displayName = 1;
+  string description = 2;
+}
+
+service ProviderService {
+  rpc List(ProviderListRequest) returns (ProviderList) {
+    option (google.api.http) = {
+      get : "/v1/tenants/{tenant}/providers"
```

Comment:
```suggestion
      get : "/v1/providers"
```

Reply:
nope, this is the endpoint for listing provider objects in the account namespace, so it should be:
"/v1/accounts/{account}/providers"

Reply:
and also for the following "get"